toolkit/geoview-compose/src/main/java/com/arcgismaps/toolkit/geoviewcompose/GeoViewScope.kt

```diff
@@ -180,6 +177,42 @@ public sealed class GeoViewScope protected constructor(private val geoView: GeoV
         isCalloutBeingDisplayed.set(false)
     }
 
+    /**
+     * Creates a Callout at the specified geographical location on the GeoView.
```

Comment:
To help differentiate this with the other `CalloutInternal()` function: 
```suggestion
     * Creates a Callout at the specified [geoElement] location on the GeoView.
```